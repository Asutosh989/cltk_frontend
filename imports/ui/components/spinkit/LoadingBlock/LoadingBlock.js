
import React from 'react';

export default class LoadingBlock extends React.Component {
	render() {
		return (
			<div className="loading-spinner loading-spinner-block">
			</div>
		);
	}
}
