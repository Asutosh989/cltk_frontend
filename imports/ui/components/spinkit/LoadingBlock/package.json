{
  "name": "LoadingBlock",
  "version": "0.0.1",
  "private": true,
  "main": "./LoadingBlock.js"
}
