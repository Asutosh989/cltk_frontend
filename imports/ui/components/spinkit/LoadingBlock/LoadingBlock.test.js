
import React from 'react';
import renderer from 'react-test-renderer';
import MuiThemeProvider from 'material-ui/styles/MuiThemeProvider';

// component:
import LoadingBlock from './LoadingBlock';

describe('LoadingBlock', () => {
	it('renders correctly', () => {

		const tree = renderer
			.create(
				<MuiThemeProvider>
					<LoadingBlock />
				</MuiThemeProvider>
			)
			.toJSON();
		expect(tree).toMatchSnapshot();
	});
});
