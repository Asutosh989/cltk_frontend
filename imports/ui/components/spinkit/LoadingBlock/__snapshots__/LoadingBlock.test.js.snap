// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoadingBlock renders correctly 1`] = `
<div
  className="loading-spinner loading-spinner-block"
/>
`;
